From 11ca50db20343d86b55eed05b3ba085f97e504d8 Mon Sep 17 00:00:00 2001
From: Shuguang Gong <shuguang.gong@intel.com>
Date: Wed, 6 May 2015 09:33:22 +0800
Subject: [PATCH 393/398] atomisp.h: add ioctl for edge enhancement

This ioctl is for EE enabling of SoC sensor

Change-Id: I48143b933f42e7c5267d62cc15df5ed1c57ceec7
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-34925
Signed-off-by: Shuguang Gong <shuguang.gong@intel.com>
Reviewed-on: https://android.intel.com/#/c/362466
---
 include/linux/atomisp.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/linux/atomisp.h b/include/linux/atomisp.h
index 91000a1..f75555c 100644
--- a/include/linux/atomisp.h
+++ b/include/linux/atomisp.h
@@ -1209,6 +1209,9 @@ struct atomisp_sensor_ae_bracketing_lut {
 #define ATOMISP_IOC_G_DEPTH_SYNC_COMP \
 	_IOR('v', BASE_VIDIOC_PRIVATE + 46, unsigned int)
 
+#define ATOMISP_IOC_S_SENSOR_EE_CONFIG \
+	_IOW('v', BASE_VIDIOC_PRIVATE + 47, unsigned int)
+
 /*
  * Reserved ioctls. We have customer implementing it internally.
  * We can't use both numbers to not cause ABI conflict.
-- 
1.9.1

